### PR TITLE
Limit the size of the track and album name received by MusicService.

### PR DESCRIPTION
**Pull request by JF002**
This should work around this bug : https://github.com/InfiniTimeOrg/InfiniTime/issues/825 and prevent heap over-allocation.

### CODE REVIEW

File: src/components/ble/MusicService.cpp [hunk 1]
```diff
@@ -125,6 +127,11 @@ void Pinetime::Controllers::MusicService::Init() {
 int Pinetime::Controllers::MusicService::OnCommand(uint16_t conn_handle, uint16_t attr_handle, struct ble_gatt_access_ctxt* ctxt) {
   if (ctxt->op == BLE_GATT_ACCESS_OP_WRITE_CHR) {
     size_t notifSize = OS_MBUF_PKTLEN(ctxt->om);
+
+    if(notifSize > MaxStringSize) {
```
Comment (Avamander) [inline src/components/ble/MusicService.cpp:131]: Missing space :stuck_out_tongue:

**Comment by Avamander** [comment]
Do you think 40 is enough, even with non-ASCII names? Or should we just look at it when it becomes an issue?

Also when truncated, maybe it should have an ellipsis at the end?

File: src/components/ble/MusicService.cpp (hunk 1 above)
Comment (JF002) [reply src/components/ble/MusicService.cpp]: Done :)

**Comment by JF002** [comment]
> Do you think 40 is enough, even with non-ASCII names? Or should we just look at it when it becomes an issue?
> 
> Also when truncated, maybe it should have an ellipsis at the end?

According to my tests, the rendering time in LVGL takes too much time with more than 40-45 characters. That's why I chose this value :)

Regarding the ellipsis : yes, why not. How would you do it? Simply replace the last 3 characters by `.`?

**Comment by Avamander** [comment]
> Regarding the ellipsis : yes, why not. How would you do it? Simply replace the last 3 characters by `.`?

If original exceeds 40, yes?

**Comment by JF002** [comment]
> If original exceeds 40, yes?

Yes, of course :)

**Comment by JF002** [comment]
Here is the result with the ellipsis :)

https://user-images.githubusercontent.com/2261652/158887933-ef806548-d8ee-48e8-9eab-4d76e67cde74.mp4

**Pull request merged**